core/primitives/piece/piece_data.hpp

```diff
@@ -16,6 +16,7 @@ namespace fc::primitives::piece {
    public:
     explicit PieceData(const std::string &path_to_file, int flags = O_RDWR);
     explicit PieceData(int &pipe_fd);
```

Comment:
please, add comment that `pipe_fd` is out parameter and is expected to be set -1 after call.

Reply:
Fixed